lib/gooddata/lcm/actions/synchronize_ldm.rb

```diff
@@ -41,43 +41,59 @@ class << self
         def call(params)
           include_ca = params.include_computed_attributes.to_b
           exclude_fact_rule = params.exclude_fact_rule.to_b
-
-          results = []
-
           client = params.gdc_gd_client
-          development_client = params.development_client
+          synchronize = []
+          results = []
+          params.synchronize.each do |segment_info|
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)